### PR TITLE
Added Development Window

### PR DESCRIPTION
**Pull request by fraher**
- Removed Redux from Application
- Added support for ReactFlow library ([ReactFlow](https://reactflow.dev/examples/drag-and-drop/))
- Removed "WeatherForecast" starter code and counter logic

### CODE REVIEW

File: fishcore.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (fraher) [inline fishcore.sln:2]: Added support for Visual Studio if a user wants to use that platform.

File: fishcore/ClientApp/.gitignore [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-.DS_Store
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
-
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (fraher) [inline fishcore/ClientApp/.gitignore:1]: Updated .gitignore to work for react/vs code/visual studio/dotnet core

File: fishcore/ClientApp/package.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (fraher) [inline fishcore/ClientApp/package.json:1]: This file contains all of the packages used by the React portion of the application

File: fishcore/ClientApp/src/components/DevelopmentWindow.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
+﻿import React, { useState, useRef } from 'react';
```
Comment (fraher) [inline fishcore/ClientApp/src/components/DevelopmentWindow.js:1]: The DevelopmentWindow currently uses the example code from [React Example](https://reactflow.dev/examples/drag-and-drop/)

File: fishcore/About.cs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace fishcore
+{
+    public class About
```
Comment (fraher) [inline fishcore/About.cs]: New class for testing how to use Fetch. The "About" class contains a single string for passing back to the front end.

File: fishcore/ClientApp/src/App.js [hunk 6]
```diff
@@ -2,23 +2,19 @@ import React, { Component } from 'react';
...
-        <Route path='/counter' component={Counter} />
-        <Route path='/fetch-data' component={FetchData} />
-        <DevelopmentWindow />
-      </Layout>
-    );
-  }
+    render() {
+        return (
+            <Layout>
+                <Route exact path='/' component={Home} />
```
Comment (fraher) [inline fishcore/ClientApp/src/App.js:15]: Updated the nav bar, removed starter project components.

File: fishcore/ClientApp/src/components/FetchAbout.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+
+export class AboutData extends Component {
```
Comment (fraher) [inline fishcore/ClientApp/src/components/FetchAbout.js]: This is the React "About" page, it calls the backend and is the component for the content that shows when you choose the page.

File: fishcore/ClientApp/src/components/Home.js [hunk 8]
```diff
@@ -1,13 +1,15 @@
...
-    return (
-      <div>
-        
-      </div>
-    );
-  }
+    render() {
+        return (
+            <div>
+                <DevelopmentWindow />
```
Comment (fraher) [inline fishcore/ClientApp/src/components/Home.js:11]: The "Home" page renders the "DevelopmentWindow" component.

File: fishcore.sln (hunk 1 above)
Comment (adricardenas7890) [reply fishcore.sln:2]: Awesome! This can be great down the line if we get a license for Visual Studio.

File: fishcore/ClientApp/src/App.js (hunk 6 above)
Comment (adricardenas7890) [reply fishcore/ClientApp/src/App.js:15]: Awesome! Nice and clean.

File: fishcore.sln (hunk 1 above)
Comment (fraher) [reply fishcore.sln:2]: Luckily we can use Visual Studio Community edition. Has a lot of useful features even at that level.

**Pull request merged**